.azure/pipelines/ci.yml

```diff
@@ -281,6 +282,14 @@ stages:
         includeForks: true
       - name: Windows_Packages
         path: artifacts/packages/
+      - name: Windows_ANCM
```

Comment:
So is it unreasonable for us to always have these binaries available?  I can just add these to my branches if we don't want them for all PRs

Reply:
Yeah, today we only upload installers in internal builds to save space, we shouldn't treat these differently